keylime/api_version.py

```diff
@@ -16,6 +17,8 @@
     "2": "2.0"
 }
 
+MINIMUM_VERSION = "1.0"
```

Comment:
Instead of hardcoding this, can we calculate it based on what is in the `VERSIONS` list?